components/brave_welcome_ui/components/select-profile/index.tsx

```diff
@@ -72,12 +72,14 @@ function SelectProfile () {
 
   const handleImportProfiles = () => {
     if (selectedProfiles.size <= 0) return
-
+    let entries: number[] = []
```

Comment:
IMO, we should handle separately when `selectedProfiles.size` is `1` by using `importData` instead of `Bulk`.
Or handle it at c++ handler.
If not, another profile will be created even if user select only one profile.

Reply:
Sure, moved bulk handler to separate class